hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/meta/SigTransactionMetadata.java

```diff
@@ -43,26 +42,25 @@
 public class SigTransactionMetadata implements TransactionMetadata {
     protected List<HederaKey> requiredKeys = new ArrayList<>();
     protected TransactionBody txn;
-    protected AccountStore store;
+    protected AccountKeyLookup keyLookup;
```

Comment:
In my understanding, an instance of `AccountKeyLookup` will usually contain a reference to the underlying data storage. If that is correct, we should not store it here. `TransactionMetadata` should not have any reference to the underlying data storage at all in my understanding. It may become archived in between and the handle method should use a different version anyway (the latest mutable state).

Reply:
Removed `AccountKeyLookup` from `SigTransactionMetadata` and used in `SigTransactionMetadataBuilder`